update.sh

```diff
@@ -0,0 +1,59 @@
+echo "Usage: $0 [option] ...
+      -u or --update  : update the version and commit to Github
+      -p or --publish : publish the package to PyPI
+      -h or --help    : help
+"
+
+for arg in "$@"
+
+do
+  if [ "$arg" == "--update" ] || [ "$arg" == "-u" ]
+  then
+      # bump the version
+      echo "Updating the package version..."
+      OLD_VER=$(cat VERSION)
+      NEW_VER=$(echo "$OLD_VER" + .01 | bc)
```

Comment:
should be able to pass a new version explicitly, in case we don't want to use the default bump